### DISCUSSION TITLE
[Account Flagged] My GitHub account has been flagged as `Spammy`

### DISCUSSION THREAD

**Discussion by pixincreate** [community/community#125292] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, I'm aware that I should be raising a ticket for account related issues, I'm left with no other option other than coming here to gain your attention and it has been 48+ hours.

And I'm aware only GitHub staff can view this.

This has happened to my other account previously and I wonder what made GitHub to reinstate my account within 12 hours back then.

I have read `code of conduct` and yes, I cannot create another account under same email. I did go through guidelines and yes, my account has been flagged `Spammy` for (violations!?) reasons that are unknown. I did raise reinstatement request (ticket number: `2805493`, `2808389`) which does not seem to help me in any way.

This is hampering my work as my account is hidden from public and no one in my Organization is able to review the pull requests that I have raised so far causing in delays that unintentional.

Any help could be helpful, thanks!

[github.com/orgs/community/discussions/125328](https://github.com/orgs/community/discussions/125328)

**Comment by cuikeyao** [comment]
My account was also flagged a month ago and it still hasn't been resolved until now. May I ask if your account has been restored?

**Reply by KiExitDispatcher** [reply]
did they help you yet?. im having same issue. i have uploaded remotely license + updated readme without timeouts so i got flagged.

**Comment by wldoui** [comment]
Hi everyone, what should I do? My account is also marked as spam, and GitHub does not remove the mark from it. I do not know what to do and how to be? :(
